tests/NuGet.IndexingTests/ResponseFormatterTests.cs

```diff
@@ -0,0 +1,484 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Lucene.Net.Search;
+using Newtonsoft.Json;
+using NuGet.Indexing;
+using NuGet.IndexingTests.TestSupport;
+using Xunit;
+
+namespace NuGet.IndexingTests
+{
+    public class ResponseFormatterTests
+    {
+
+        [Theory]
+        [MemberData(nameof(V2CountResultData))]
+        public void WriteV2CountResultTest(int totalHits, string expected)
+        {
+            StringBuilder sb = new StringBuilder();
```

Comment:
![Var all the things](https://cloud.githubusercontent.com/assets/485230/18577353/68ec2fba-7be9-11e6-9415-48709d04fb07.jpg)